fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/FDBRecordStoreBase.java

```diff
@@ -1511,6 +1512,24 @@ default int getPipelineSize(@Nonnull PipelineOperation pipelineOperation) {
         @Nonnull
         BaseBuilder<M, R> setMetaDataStore(@Nullable FDBMetaDataStore metaDataStore);
 
+        /**
+         * Set the local version of the record meta-data file descriptor. Users may have a local meta-data file descriptor
+         * containing a newer version of record types or indexes that follows the
+         * <a href="https://foundationdb.github.io/fdb-record-layer/SchemaEvolution.html">Record Layer guidelines on schema evolution</a>.
+         * This will only be used if {@link #setMetaDataProvider} and {@link #setMetaDataStore} are also called.
```

Comment:
I think it's reasonable to only use this if `setMetaDataStore` is also called. I think requiring the user *also* set `setMetaDataProvider` is unnecessary (as that just provides a seed meta-data in the case where `MetaDataStore` is set).

Reply:
That wasn't necessarily my suggestion.

Reply:
Have dropped the method.